deploy/k8s/charts/trustification/templates/services/spog/api/030-Deployment.yaml

```diff
@@ -32,12 +32,14 @@ spec:
           {{- include "trustification.application.container" $mod | nindent 10 }}
           {{- include "trustification.application.infrastructure.probes" $mod | nindent 10 }}
 
-          command: ["/trust"]
+          command: [ "/trust" ]
           args:
             - "spog"
             - "api"
             - "-p"
             - "8080"
+            - "--db-storage-base"
+            - "{{ include "trustification.preferences.db.path" $mod}}"
```

Comment:
What will happen when the user decides to scale up this deployment?

Reply:
That's a good point.

With this approach based on SQLite it has been accepted, like it happens for the collectorist component, the "compromise" of losing the ability to scale up these components based on the considerations that:

- the performance tests gave good enough results with the current default configuration
- there's no plan, considered the trustification's lifespan, to invest further efforts in testing the scaling up of the current architecture components

It's very opinionated, we all know, but I hope at least this makes sense.

Reply:
We should just be clear about that in the documentation.